src/encoded/audit/biosample.py

```diff
@@ -178,15 +178,27 @@ def audit_biosample_part_of_consistency(value, system):
 # utility functions
 
 def is_part_of(term_id, part_of_term_id, ontology):
```

Comment:
Is there a test for this?  If not, where we would put it?  Would be good to add one.

Reply:
There's no test for the is_part_of helper function, although there are some for the audit itself. I don't know how to access the ontology from the test app, though

Reply:
I think @p-sud you can mock-up ontology for the test purposes